src/mcu.cpp

```diff
@@ -1351,6 +1351,26 @@ int main(int argc, char *argv[])
             {
                 resetType = ResetType::GM_RESET;
             }
+            else if (!strcmp(argv[i], "-h") || !strcmp(argv[i], "-help") || !strcmp(argv[i], "--help"))
+            {
+                printf("Usage: %s [options]\n", argv[0]);
```

Comment:
not sure if argv[0] is right here, it may have full path. simply `nuked-sc55` might be better

Reply:
we probably want to do something similar to what GCC does, where we print out the executable's name instead of hard coding.